tests/execute/upgrade/main.fmf

```diff
@@ -26,3 +26,7 @@ tier: 4
     summary: Plan and upgrade path in the local repo
     framework: shell
     test: cd local && tmt run -vvv --remove plan --name plan
+
+/skip-tests:
+    summary: Skip tests before and/or after upgrade
+    test: ./skip-tests.sh
```

Comment:
Should be tagged with `provision-only` and `provision-virtual` to be run in the `provision` which should fix tests failing because `libvirtd` is not started.